### PR TITLE
(PUP-8314) add decorations for raise ArgumentError strings part 3

### PR DESCRIPTION
**Pull request by ericldelaney**

### CODE REVIEW

File: lib/puppet/pops/evaluator/epp_evaluator.rb [hunk 1]
```diff
@@ -4,15 +4,18 @@ class Puppet::Pops::Evaluator::EppEvaluator
...
+          { class_name: epp_source.class }
     end
 
     # Parse and validate the source
     parser = Puppet::Pops::Parser::EvaluatingParser::EvaluatingEppParser.new
     begin
       result = parser.parse_string(epp_source, 'inlined-epp-text')
     rescue Puppet::ParseError => e
-      raise ArgumentError, "inline_epp(): Invalid EPP: #{e.message}"
+      #TRANSLATORS 'inline_epp()' is a method name and 'epp' refers to 'Embedded Puppet (EPP) template' and should not be translated
```
Comment (melissa) [inline lib/puppet/pops/evaluator/epp_evaluator.rb]: for clarity, I think `'epp'` in this note should be `'EPP'`

File: lib/puppet/pops/evaluator/epp_evaluator.rb [hunk 2]
```diff
@@ -41,7 +45,8 @@ def self.epp(scope, file, env_name, template_args = nil)
     begin
       result = parser.parse_file(template_file)
     rescue Puppet::ParseError => e
-      raise ArgumentError, "epp(): Invalid EPP: #{e.message}"
+      #TRANSLATORS 'epp()' is a method name and 'epp' refers to 'Embedded Puppet (EPP) template' and should not be translated
```
Comment (melissa) [inline lib/puppet/pops/evaluator/epp_evaluator.rb]: same as above with the capital `'EPP'`

File: lib/puppet/pops/evaluator/epp_evaluator.rb [hunk 3]
```diff
@@ -55,13 +60,16 @@ def self.evaluate(parser, func_name, scope, use_global_scope_only, parse_result,
 
     body = parse_result.body
     unless body.is_a?(Puppet::Pops::Model::LambdaExpression)
-      raise ArgumentError, "#{func_name}(): the parser did not produce a LambdaExpression, got '#{body.class}'"
+      raise ArgumentError, _("%{function_name}(): the parser did not produce a LambdaExpression, got '%{class_name}'") %
```
Comment (melissa) [inline lib/puppet/pops/evaluator/epp_evaluator.rb]: `LambdaExpression` looks like a class name here

File: lib/puppet/pops/evaluator/epp_evaluator.rb [hunk 4]
```diff
@@ -55,13 +60,16 @@ def self.evaluate(parser, func_name, scope, use_global_scope_only, parse_result,
...
+          { function_name: func_name, class_name: body.class }
     end
     unless body.body.is_a?(Puppet::Pops::Model::EppExpression)
-      raise ArgumentError, "#{func_name}(): the parser did not produce an EppExpression, got '#{body.body.class}'"
+      raise ArgumentError, _("%{function_name}(): the parser did not produce an EppExpression, got '%{class_name}'") %
```
Comment (melissa) [inline lib/puppet/pops/evaluator/epp_evaluator.rb]: `EppExpression` also looks like a class name

File: lib/puppet/pops/evaluator/epp_evaluator.rb [hunk 5]
```diff
@@ -55,13 +60,16 @@ def self.evaluate(parser, func_name, scope, use_global_scope_only, parse_result,
...
+          { function_name: func_name, class_name: body.body.class }
     end
     unless parse_result.definitions.empty?
-      raise ArgumentError, "#{func_name}(): The EPP template contains illegal expressions (definitions)"
+      #TRANSLATORS 'epp' refers to 'Embedded Puppet (EPP) template'
```
Comment (melissa) [inline lib/puppet/pops/evaluator/epp_evaluator.rb]: same as above about capital `'EPP'`

File: lib/puppet/pops/evaluator/epp_evaluator.rb (hunk 1 above)
Comment (ericldelaney) [reply lib/puppet/pops/evaluator/epp_evaluator.rb]: Agreed

File: lib/puppet/pops/evaluator/epp_evaluator.rb (hunk 2 above)
Comment (ericldelaney) [reply lib/puppet/pops/evaluator/epp_evaluator.rb]: Agreed

File: lib/puppet/pops/evaluator/epp_evaluator.rb (hunk 3 above)
Comment (ericldelaney) [reply lib/puppet/pops/evaluator/epp_evaluator.rb]: changed

File: lib/puppet/pops/evaluator/epp_evaluator.rb (hunk 4 above)
Comment (ericldelaney) [reply lib/puppet/pops/evaluator/epp_evaluator.rb]: Yeah, changed

File: lib/puppet/pops/evaluator/epp_evaluator.rb (hunk 5 above)
Comment (ericldelaney) [reply lib/puppet/pops/evaluator/epp_evaluator.rb]: Agreed

**Pull request merged**